fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/RdfLexicon.java

```diff
@@ -155,6 +156,9 @@
     public static final Property MEMBER_SUBJECT =
             createProperty(LDP_NAMESPACE + "MemberSubject");
 
+    public static final Set<Resource> SUPPORTED_INTERACTION_MODELS = new HashSet<>(Arrays.asList(
```

Comment:
There seems to be another like this on line 268 `INTERACTION_MODELS`

Reply:
I've made a change to address this, let me know if it looks okay https://github.com/fcrepo4/fcrepo4/pull/1486/commits/45eeb8900f9e84f44ceb9e45006a313dc0410384